frontend/packages/console-shared/src/constants/common.ts

```diff
@@ -61,6 +61,7 @@ export enum FLAGS {
   PROMETHEUS = 'PROMETHEUS',
   CHARGEBACK = 'CHARGEBACK',
   OPENSHIFT = 'OPENSHIFT',
+  MONITORING = 'MONITORING',
```

Comment:
@spadgett @christianvogt Are there any conditions for adding new flags?

Reply:
Could we expect that monitoring is available "not" or "at all"? I mean: Should I add separate flags for the SERVER_FLAGS `prometheusBaseURL` and `prometheusTenancyBaseURL` ??

Reply:
I can only guess that you'd have none or both. But I'd have to defer to someone more knowledgable on this.